web/renderer/components/ConnectionsAndDatabases/DatabaseItem.tsx

```diff
@@ -60,7 +60,7 @@ export default function DatabaseItem({
   const dbName = excerpt(db, 32);
   if (db === currentDatabase && conn.name === currentConnection.name) {
     return (
-      <span className={css.dbItem}>
+      <span className={css.dbItem} data-cy={`database-${dbName}`}>
```

Comment:
Should probably be distinguished from below?
```suggestion
      <span className={css.dbItem} data-cy={`current-database-${dbName}`}>
```